.babelrc

```diff
@@ -1,16 +1,32 @@
 {
-  "presets": [
-    ["es2015", {
-      "loose": true,
-      "modules": false
-    }],
-    "stage-0"
-  ],
   "plugins": [
     "transform-flow-strip-types"
   ],
   "env": {
+    "production": {
+      "presets": [
+        ["es2015", {
+          "loose": true,
+          "modules": false
+        }],
+        "stage-0"
+      ]
+    },
+    "development": {
```

Comment:
I wonder if we should just combine `development` and `test, since they're the same? Maybe just `production` and `development?